content/5.resources/3.blockexplorers.md

```diff
@@ -31,3 +31,7 @@ Info
 ### Nodes guru
 
 <a href="https://explorers.guru/" target="_blank">Nodes.Guru</a> (Archway not available yet)
+
+### ScanRun
+- <a href="https://scanrun.io/archway-testnet" target="_blank"> Constantine </a>
+- <a href="https://scanrun.io/archway" target="_blank"> Mainnet </a> (Not available yet)
```

Comment:
Thanks for adding Archway to your block explorer and submitting this PR. Regarding links that are not yet available, we have decided to remove those links as shown in this PR: https://github.com/archway-network/archway-docs/pull/239. Therefore, I suggest removing the mainnet link above, after which I can proceed with the merge.

```suggestion

```

Reply:
Done